### PR TITLE
added kinesis_agent role and included in formplayer

### PR DESCRIPTION
**Pull request by knguyen142**
https://trello.com/c/prhWvkky/37-hipaa-logging-improvements

- Creates a `kinesis_agent` role
  - creates a tmp dir, git pulls source code, installs from setup script, then removes directory
  - generates config file and enables service
- Adds `kinesis_agent` role to deploy_formplayer.yml
- Requires a `kinesis_flow` nonempty list to be defined to run
- Can be added to prod once everything set up 

Screen Shot 2020-08-11 at 6 46 46 PM

##### ENVIRONMENTS AFFECTED

Staging

### CODE REVIEW

File: src/commcare_cloud/ansible/roles/kinesis_agent/templates/agent.json.j2 [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "cloudwatch.emitMetrics": true,
+  "kinesis.endpoint": "{{ kinesis_endpoint }}",
+  "firehose.endpoint": "{{ firehose_endpoint }}",
+
+  "flows": [
+    {% for flow in kinesis_flows -%}
+    {
+      "filePattern": "{{ flow.file_pattern }}",
+      "deliveryStream": "{{ flow.delivery_stream }}"
+    }
```
Comment (dannyroberts) [inline src/commcare_cloud/ansible/roles/kinesis_agent/templates/agent.json.j2:11]: nit: if there were more than one then you'd need to insert a comma here on all but the last one

**Comment by knguyen142** [comment]
Thanks @dannyroberts. Do you want me to add the `kinesis_flows` var for other envs too in this PR (production, india, swiss?)

**Comment by dannyroberts** [comment]
Yes please, for india and production only

**Comment by knguyen142** [comment]
@dannyroberts Awesome, can you run the terrraform kinesis stream on India? I see it for prod but not India.

**Comment by knguyen142** [comment]
India (with [region endpoints](https://github.com/dimagi/commcare-cloud/pull/4077/files#diff-d1da1be54f3239e858cf9c57cd5d6eacR246-R247)) and Prod added

I ran with 
`cchq staging ansible-playbook deploy_formplayer.yml --branch=kev/formplayer-install-kinesis-agent --tags=kinesis-agent`

So I guess it will be 
`cchq production ansible-playbook deploy_formplayer.yml --tags=kinesis-agent`

Let me know when you've run it and I can verify the logs are getting shipped to S3.

**Pull request merged**